analytic_engine/src/instance/flush_compaction.rs

```diff
@@ -914,6 +914,7 @@ impl SpaceStore {
                 sst_type: table_data.sst_type,
                 read_batch_row_num: table_options.num_rows_per_row_group,
                 reverse: false,
+                just_once: true,
```

Comment:
just_once seems awkward, how about introduce a `ReadType`
```
enum ReadType {
   Compact,
   Select,
   ...
}
impl ReadType {
  fn cache_sst(self) -> bool {

  }
  fn require_bloom_filter(&self) -> bool {
  }
}
```

Reply:
Replacing the boolean value with an enum sounds good, but it may not be a good pattern in terms of the two types of that. The sst reader is the dependency of`Query` and `Compaction` module, which means it should have no knowledge of its callers. Otherwise, the more read types are needed when more dependencies are introduced.

Maybe we should replace the `ReadType` with `ReadFrequency`:
```
enum ReadFrequency {
    Once,
    Repetitive,
}
```

Reply:
Sounds good.